### ISSUE TITLE
Failed to load the jupyterlab-git server extension

### ISSUE THREAD

**Issue by JupyterPS** [jingsta/powershell-playground#1]
When I run this \forked\ repo I get the following message as it finishes building

Failed to load the jupyterlab-git server extension
gitcommand not found - please ensure you have Git > 2 installed

Please help if possible

I have taken TOC out because it is too hard to get throug a build and is easier to load the extension later

**Comment by jingsta** [comment]
Gonna try redoing this

**Comment by JupyterPS** [comment]
Thank you......while you are there I hope you are able to recognize/solve the following question I have laid out on Microsofts Q&A site:
Cmdlet out-display in Powershell stopped working

Making a rebuild of my GITHUB repository running DOCKER in BINDER to get a fresh NOTEBOOK in JUPYTERLAB
I had to use a new version of .NETS interactive tool:

global Microsoft.dotnet-interactive --version 1.0.155302 --add-source "https://dotnet.myget.org/F/dotnet-try/api/v3/index.json";

In the new notebook I noticed that: The cmdlet Out-display and Out-Display -CustomMimeType stopped working handling HTML code
PS: You used version 1.0.131806 in your first dockerfile. Do you think the new version could affect the above question?

**Comment by JupyterPS** [comment]
Thank you......while you are there I hope you are able to recognize/solve the following question I've laid out on Microsofts Q&A site:
____________________________________________________

Cmdlet out-display in Powershell stopped working

Making a rebuild of my GITHUB repository running DOCKER in BINDER to get a fresh NOTEBOOK in JUPYTERLAB
I had to use a new version of .NETS interactive tool:

global Microsoft.dotnet-interactive --version 1.0.155302 --add-source "https://dotnet.myget.org/F/dotnet-try/api/v3/index.json";

In the new notebook I noticed that: The cmdlet Out-display and Out-Display -CustomMimeType stopped working handling HTML code
_______________________________________________________

PS: You used version 1.0.131806 in your first dockerfile. Do you think the new version could affect the above question? 🙂
________________________________
Fra: Jing <notifications@github.com>
Sendt: 6. februar 2021 05:46
Til: jingsta/jupyter-playground <jupyter-playground@noreply.github.com>
Cc: Henning Reckey <henningreckey@hotmail.com>; Author <author@noreply.github.com>
Emne: Re: [jingsta/jupyter-playground] Failed to load the jupyterlab-git server extension (#1)


Gonna try redoing this

**Comment by jingsta** [comment]
Yeah it happened to me too! I think it is the new version but I am trying to figure it out. It all happened when I tried adding rust and trying to get the github stuff working. I think I am almost there. 

What do you use outdisplay for

**Comment by JupyterPS** [comment]
So far I have covered the following themes, since I first ran into your Powershell tutorials 1-15 and I still remember your line:

                            #### The magic of the `Out-Display` cmdlet   ==>    in your HTML.ipynb

Now I'm going to see what HTML/CSS can do to live Jupyterlab up a little 😊
-----------------------------------------------------------------------------------
  PYTHON       *   POWERSHELL   JUPYTERRLAB  (coming up next) HTML/CSS
-  Matplotlib  *       - Xplot           - Github
- Numpy         *                              - Git
- Pyplot          *                              - Binder
- Scipy            *                              - Docker
- Pandas        *                               - Markdown
- Plotly           *                              - Latex
------------------------------------------------------------------------------------

Thank you again for your time and it has been an exiting journey

________________________________
Fra: Jing <notifications@github.com>
Sendt: 8. februar 2021 18:22
Til: jingsta/jupyter-playground <jupyter-playground@noreply.github.com>
Cc: Henning Reckey <henningreckey@hotmail.com>; Author <author@noreply.github.com>
Emne: Re: [jingsta/jupyter-playground] Failed to load the jupyterlab-git server extension (#1)


Yeah it happened to me too! I think it is the new version but I am trying to figure it out. It all happened when I tried adding rust and trying to get the github stuff working. I think I am almost there.

What do you use outdisplay for

**Comment by jingsta** [comment]
I think the latest one works if you build it. I have to remove Jupyter lab and install Jupyterlab-git to get it working. It works locally but not on mybinders.com.
